chore: [FFM-1907]: README Updates for M1 Macs (#8748)
harness/harness-core-ui: docs/M1_macbooks.md

## Additional Installation Steps for M1 based Macbooks

> When following the ReadMe there are additional steps required for users with a M1 MacBook. Please see below:

Reviewer: ```suggestion
Additional steps required prior to installation for users with a M1 Mac. Please see below:
```
